docs-src/references/server-api.md

```diff
@@ -0,0 +1,108 @@
+---
+id: server-api
+title: Temporal Server API
+description: You can interact with Temporal Server via one of its APIs.
+tags:
+  - reference
+---
+
+While it's usually easiest to interact with [Temporal Server](/concepts/what-is-the-temporal-server) via a [Client SDK](/concepts/what-is-a-temporal-client) or the [Temporal CLI](https://docs.temporal.io/cli), you can also use one of its APIs:
+
+- [gRPC API](#grpc-api)
+- [HTTP API](#http-api)
+
+## gRPC API
+
+Our Client and Worker SDKs use the gRPC API. The API reference is located here:
+
+[`api-docs.temporal.io`](https://api-docs.temporal.io/)
+
+### Use with code
+
+Usually you interact with the API via high-level methods like `client.workflow.start()`. However, Client SDKs also expose the underlying gRPC services. For instance, the TypeScript SDK has:
+
+- WorkflowService: [`Client.connection.workflowService`](https://typescript.temporal.io/api/classes/client.Connection#workflowservice)
+- OperatorService: [`Client.connection.operatorService`](https://typescript.temporal.io/api/classes/client.Connection/#operatorservice)
+- HealthService: [`Client.connection.healthService`](https://typescript.temporal.io/api/classes/client.Connection/#healthservice)
+
+If you're not using an SDK Client (rare), you can generate gRPC client stubs by:
+
+- Adding [`temporalio/api`](https://github.com/temporalio/api) (repo with the protobuf files) as a git submodule or subtree inside your repo
+- Generating code in [your language](https://grpc.io/docs/languages/)
+
+### Use manually
+
+To query the API manually via command line or a GUI, first:
+
+- Run [`temporal server start-dev`](/cli/#starting-the-temporal-server)
+- Clone this repo:
+
+  ```sh
+  git clone https://github.com/temporalio/api.git
+  cd api
+  ```
+
+#### With command line
+
+Install [`evans`](https://github.com/ktr0731/evans#installation).
+
+```
+cd /path/to/api
+evans --proto temporal/api/workflowservice/v1/service.proto --port 7233
+```
+
+#### With a GUI
+
+- Install [BloomRPC](https://github.com/bloomrpc/bloomrpc#installation).
+- Open the app
+- Select "Import Paths" button on the top-left and enter the path to the cloned repo: `/path/to/api`
+- Select the "Import protos" + button and select this file:
+
+  `/path/to/api/temporal/api/workflowservice/v1/service.proto`
+
+- A list of methods should appear in the sidebar. Select one.
+- Edit the JSON in the left pane.
+- Hit `Cmd-Enter` or click the play button to get a response from the server on the right.
+
+![ListWorkflowExecutions](/img/ListWorkflowExecutions.png)
+
+One downside compared to [command line](#with-command-line) is it doesn't show enum names, just numbers like `"task_queue_type": 1`.
+
+![DescribeTaskQueue](/img/DescribeTaskQueue.png)
+
+## HTTP API
```

Comment:
This section should not be published at all IMO. The HTTP API may not even come to fruition (nobody approves it as is and there are dissenting voices), and we should not say the API exposed by ui-server should be relied on programmatically by users (even with may-have-breaking-changes caveats).